### PR TITLE
feature-PAJ-5-user-services

### PR DESCRIPTION
**Pull request by jaroslaw-majka**
Scripts clean-up for better readability
UserServices class created which maintains (for now) user login and prints admin menu.

### CODE REVIEW

**Review by ewelina-jeschke — APPROVED** [review]
approve

File: users/user_classes/class_receptionist.py [hunk 1]
```diff
@@ -42,6 +44,3 @@ def show_doctor_appointment(self):
 
     def remove_doctor_appointment(self):
         print('Wizyta lekarska została usunięta.')
```
Comment (ewelina-jeschke) [inline users/user_classes/class_receptionist.py:46]: za duzo spacji

File: users/user_menus/admin_menu.py [hunk 2]
```diff
@@ -65,8 +66,9 @@ def validate_decision(decision):
         raise Exception(f"Liczba {decision} jest niedozwolona!")
 
 
-while True:
```
Comment (ewelina-jeschke) [inline users/user_menus/admin_menu.py:68]: literowka

**Review by ewelina-jeschke — CHANGES_REQUESTED** [review]

File: users/user_classes/class_receptionist.py (hunk 1 above)
Comment (jaroslaw-majka) [reply users/user_classes/class_receptionist.py:46]: fixed

File: users/user_menus/admin_menu.py (hunk 2 above)
Comment (jaroslaw-majka) [reply users/user_menus/admin_menu.py:68]: fixed

**Pull request merged**